frameworks/projects/MXRoyale/src/main/royale/mx/controls/listClasses/IListItemRenderer.as

```diff
@@ -0,0 +1,85 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+//  Licensed to the Apache Software Foundation (ASF) under one or more
+//  contributor license agreements.  See the NOTICE file distributed with
+//  this work for additional information regarding copyright ownership.
+//  The ASF licenses this file to You under the Apache License, Version 2.0
+//  (the "License"); you may not use this file except in compliance with
+//  the License.  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+package mx.controls.listClasses
+{
+	COMPILE::SWF {
+		import flash.events.IEventDispatcher;
```

Comment:
This doesn't seem right.  I think it can be org.apache.royale.events.IEventDispatcher even for SWF

Reply:
I looked at the previous samples to do for the older flash classes and used the same way Alex. If it is not the right way, ok i am going to change it.